src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -14,21 +22,93 @@ export const ARShowScreen = ({ navigation, route }) => {
   }, []);
 
   const parser = async () => {
-    await objectParser({ item: data[index], setObject });
+    // TODO: the `onPress` prop is just for testing
+    await objectParser({ item: data[index], setObject, onPress: () => navigation.goBack() });
+
     setIsLoading(false);
   };
 
-  if (isLoading || !object) return <LoadingSpinner loading />;
+  if (isLoading || !object || !object.vrx) return <LoadingSpinner loading />;
 
   return (
-    <SafeAreaViewFlex>
-      <RegularText>AR Show Screen</RegularText>
-      <RegularText onPress={() => navigation.goBack()}>Zurück</RegularText>
-    </SafeAreaViewFlex>
+    <>
+      <ViroARSceneNavigator
+        autofocus
+        initialScene={{
+          scene: AugmentedRealityView
+        }}
+        viroAppProps={{ object }}
+        style={styles.arSceneNavigator}
+      />
+      <TouchableOpacity
+        style={styles.backButton}
+        onPress={() => {
+          /*
+          to solve the Android crash problem, you must first remove the 3D object from the screen. 
+          then navigation can be done.
+          */
+          setObject();
+          navigation.goBack();
+        }}
+      >
+        <Icon.Close color={colors.surface} />
+      </TouchableOpacity>
+    </>
   );
 };
 
-const objectParser = async ({ item, setObject }) => {
+const AugmentedRealityView = ({ sceneNavigator }) => {
+  const [rotation, setRotation] = useState([0, 0, 0]);
+  const [position, setPosition] = useState([0, -1, -5]);
+
+  const { object } = sceneNavigator.viroAppProps;
+
+  const moveObject = (newPosition) => {
+    setPosition(newPosition);
+  };
+
+  const rotateObject = (rotateState, rotationFactor) => {
+    let newRotation = [rotation[0], rotation[1] - rotationFactor, rotation[2]];
+
+    if (rotateState === 2) {
+      setRotation(newRotation);
+
+      return;
+    }
+  };
+
+  const handleLoadStart = () => {
+    console.warn('OBJ loading has started');
+  };
+  const handleLoadEnd = () => {
+    console.warn('OBJ loading has finished');
+  };
+  const handleError = (event) => {
+    console.warn('OBJ loading failed with error: ' + event.nativeEvent.error);
```

Comment:
these `warn`s needs to be removed before merge. we should think of some loading spinner or information for the user to hint for a waiting time

Reply:
Yes, these give us information about the loading of the object. It's in log form for now. It can be replaced with an `indicator` in the future.